src/main/java/org/opentripplanner/middleware/connecteddataplatform/ConnectedDataManager.java

```diff
@@ -141,7 +141,7 @@ public void scheduleRecurringJob() {
             Scheduler.scheduleJob(
                 new TripHistoryUploadJob(),
                 initialDelayMillis,
-                CONNECTED_DATA_PLATFORM_TRIP_HISTORY_UPLOAD_JOB_FREQUENCY_IN_MINUTES,
+                CONNECTED_DATA_PLATFORM_TRIP_HISTORY_UPLOAD_JOB_FREQUENCY_IN_MINUTES * 60000L, // milliseconds
```

Comment:
I might be missing something here, but would it not be better to change `TimeUnit.MILLISECONDS` to `TimeUnit.MINUTES` instead?

Reply:
Wait. That would mess the initial delay up! Coffee brb.